### PR TITLE
[WIP]Zlib cms cloudflare new references

### PR DESCRIPTION
**Pull request by oshadura**

### CODE REVIEW

File: root/io/filemerger/execTestMultiMerge.C [hunk 1]
```diff
@@ -92,18 +92,25 @@ int execTestMultiMerge()
...
-   result += testMergedFile("mzlibfile1-4.root",106,4978+ lz4default*841 + kIs32bits*2 - kIs32bits*lz4default*16, kIs32bits ? 2 : 0);
+   result += testMergedFile("mzlibfile1-4.root",106,4978 + cloudflarezlib*7 + lz4default*841 + kIs32bits*2 - kIs32bits*lz4default*16, kIs32bits ? 2 : 0);
    result += testSimpleFile("hsimple.root",25000,expectedcomplevel,414668 + lz4default*104060 + kIs32bits*2 + kIs32bits*lz4default*19, kIs32bits ? (12 + fastMath*10) : (8 + fastMath*10));
    result += testSimpleFile("hsimple9.root",25000,9,432268 + lz4default*86230 + kIs32bits*10 - kIs32bits*lz4default*8,4 + fastMath*27);
-   result += testSimpleFile("hsimple101.root",25000,101,414856 + lz4default*1667, kIs32bits ? 12 : (3 + fastMath*14));
-   result += testSimpleFile("hsimple106.root",25000,106,432377 + lz4default*1931 + kIs32bits*4,3 + fastMath*20);
-   result += testSimpleFile("hsimple109.root",25000,109,432278 + lz4default*1931 + kIs32bits*10,3 + fastMath*28);
+   result += testSimpleFile("hsimple101.root",25000,101,414856 + cloudflarezlib*28627 + lz4default*1667, kIs32bits ? 12 : (3 + fastMath*14));
```
Comment (Axel-Naumann) [inline root/io/filemerger/execTestMultiMerge.C]: Does this mean that CFzlib is compressing 28627/414856=7% worse than traditional zlib in this case? I'm not saying it's a problem for comp level 1, I'm just asking :-)

Comment (oshadura) [reply root/io/filemerger/execTestMultiMerge.C]: Hi @axel, I was surprised today too, let me recheck and come back to you :)

**Comment by oshadura** [comment]
Failure in a sl6 build was caused by infrastructure failure.

**Comment by oshadura** [comment]
@Axel-Naumann all test failures are unrelated to changes..

**Pull request closed without merging**